server/files/entrypoint_nginx.sh

```diff
@@ -160,6 +160,8 @@ enforce_misp_data_permissions(){
     echo "... chmod -R 0770 directories /var/www/MISP/app/Config" && find /var/www/MISP/app/Config -not -perm 770 -type d -exec chmod 0770 {} +
     # We make configuration files read only
     echo "... chmod 600 /var/www/MISP/app/Config/{config,database,email}.php" && chmod 600 /var/www/MISP/app/Config/{config,database,email}.php
+    # Satisfy MISP diagnostics. Silently fails if file is not present
+    chmod -f 600 /var/www/MISP/.git/ORIG_HEAD
```

Comment:
Is this needed after merging the PR in MISP?

Also, aren't we doing already in the Dockerfile? Couldn't we chmod there?

Reply:
I tried Dockerfile chmod but the entrypoint_nginx.sh chmods prior to this make this file unwriteable, so I just added this single chmod rather than mess with the prior entrypoint commands.

Reply:
Proposal: should we change this line and the line in the Docker file to `touch && chmod` and change this comment to "do the action again because...etc". I would like to somehow make it explicit that these two lines do in theory the very same thing, and we need it in the both places because reasons.

Reply:
Actually, this is just a sign I'm operating on too much caffeine lately. The chmod is of course overridden in the COPY step in the target container, not all the find steps in the entrypoint that don't actually target the .git directory in any way. I'll move this touch/chmod bit to the same step as dist copy.